### PR TITLE
fix: solace bindings - typo "msvVpn" should be "msgVpn"

### PR DESCRIPTION
**Pull request by 195858**
https://github.com/asyncapi/bindings/issues/153

**Description**

Fixed typo in Solace server binding JSON schema

**Related issue(s)**
Resolves
https://github.com/asyncapi/bindings/issues/153

### CODE REVIEW

**Comment by 195858** [comment]
2nd commit fixes this issue 

https://github.com/asyncapi/bindings/issues/155

**Review by damaru-inc — DISMISSED** [review]

File: solace/json_schemas/server.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     }
   },
   "properties": {
-    "msvVpn": {
```
Comment (KhudaDad414) [inline solace/json_schemas/server.json:14]: Hey @195858, It would be great if you could increment the binding version as well.

Comment (195858) [reply solace/json_schemas/server.json:14]: Bumped the version number to 0.2.1

**Review by damaru-inc — APPROVED** [review]

**Comment by char0n** [comment]
Hi everybody,

I was wondering why are we increasing the binding version to `0.2.1`. As the markdown file is the source of truth for the binding, nothing has been changed in that file. This PR only addresses the fix in accompanied JSON Schema which is only supplement of the spec.

**Comment by VisualBean** [comment]
> Hi everybody,
> 
> I was wondering why are we increasing the binding version to `0.2.1`. As the markdown file is the source of truth for the binding, nothing has been changed in that file. This PR only addresses the fix in accompanied JSON Schema which is only supplement of the spec.

I agree, if it doesn't change the actual spec, then the version should not change. (Also renaming a property is a breaking change, so it would deem a major version increment, not a patch version increment).

**Pull request closed without merging**